app/tests/spec/lib/oauth-errors.js

```diff
@@ -60,19 +60,30 @@ function (chai, OAuthErrors) {
     });
 
     describe('toInterpolationContext', function () {
-      it('context returns object', function () {
+      it('returns an empty object by default', function () {
         var result = OAuthErrors.toInterpolationContext(OAuthErrors.toError('UNKNOWN_CLIENT'));
         assert.equal(Object.keys(result).length, 0);
       });
 
-      it('context returns param', function () {
+      it('`MISSING_PARAMETER` returns object w/ `param`', function () {
         var err = OAuthErrors.toError('MISSING_PARAMETER');
         err.param = 'param';
         assert.equal(OAuthErrors.toInterpolationContext(err).param, 'param');
       });
 
-      it('context catches data exception', function () {
-        assert.equal(Object.keys(OAuthErrors.toInterpolationContext(undefined)).length, 0);
```

Comment:
Not really, the test's intent was to catch malformed server responses and ensure exceptions decoding server responses aren't propagated. I renamed the test to `malformatted server responses are handled gracefully`, which should be `malformed`. I can add an `undefined` input test if you think it's worthwhile.